packages/spor-theme-react/src/components/media-controller-button.ts

```diff
@@ -0,0 +1,132 @@
+import { createMultiStyleConfigHelpers } from "@chakra-ui/react";
+import { anatomy } from "@chakra-ui/theme-tools";
+import { getBoxShadowString } from "../utils/box-shadow-utils";
+import { focusVisible } from "../utils/focus-utils";
+
+const parts = anatomy("media-controller-button").parts("container", "icon");
+const helpers = createMultiStyleConfigHelpers(parts.keys);
+
+const config = helpers.defineMultiStyleConfig({
+  baseStyle: {
+    container: {
+      fontSize: 30,
+      transitionProperty: "common",
+      transitionDuration: "fast",
+      borderRadius: "round",
+      appearance: "none",
+      background: "transparent",
+      border: "none",
+      cursor: "pointer",
+      display: "flex",
+      padding: 1,
+      alignSelf: "center",
+      color: "darkTeal",
+    },
+    icon: {
+      flex: "0 0 auto",
+      display: "block",
+      width: "1em",
+      height: "1em",
+    },
+  },
+  variants: {
+    play: {
+      container: {
+        padding: 0,
+        _hover: {
+          color: "night",
+        },
+        _active: {
+          color: "primaryGreen",
+        },
+        ...focusVisible({
+          focus: {
+            position: "relative",
+            outline: "none",
+            _after: {
```

Comment:
Her måtte jeg bruke et pseudoelement for å få stylet play / pause ikonet 😬 Vel vel, det funker i alle fall!